src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -222,7 +222,14 @@ export class ArtworkApp extends React.Component<Props> {
             </Col>
           </Row>
 
-          <div id="lightbox-container" />
+          <div
+            id="lightbox-container"
+            style={{
+              position: "absolute",
+              top: 0,
+              zIndex: 1100, // over top nav
+            }}
```

Comment:
Fixes long standing issue where when a user views an artwork in full screen, the top nav bar flops out due to z-index and position.